### PR TITLE
Dont update post count if number on scratch is higher than scratchdb's count

### PR DESCRIPTION
**Pull request by FunctionalMetatable**
### Changes
Stops erroring in console if scratchdb is down

### CODE REVIEW

File: addons/exact-count/forum.js [hunk 1]
```diff
@@ -2,10 +2,13 @@ export default async function ({ addon, global, console }) {
   while (true) {
     let userbox = await addon.tab.waitForElement(".postleft dl", { markAsSeen: true });
     if (userbox.childNodes[6] && userbox.childNodes[6].nodeValue.includes("+")) {
+      const postCountReal = Number(/(\d\d?\d?\d?)\+? posts?/.exec(userbox.childNodes[6].nodeValue)[1]);
```
Comment (TheColaber) [inline addons/exact-count/forum.js:5]: `posts` would need to be translated...

Comment (mxmou) [reply addons/exact-count/forum.js:5]: Most of the forums isn't translated.

Comment (FunctionalMetatable) [reply addons/exact-count/forum.js:5]: > 
> 
> `posts` would need to be translated...

Its never translated. I even checked by switching to a different language

Comment (apple502j) [inline addons/exact-count/forum.js:5]: `(\d+)\+? posts?` ?

Comment (FunctionalMetatable) [reply addons/exact-count/forum.js]: fixed

**Pull request merged**